include/simdjson/generic/ondemand/token_iterator.h

```diff
@@ -112,7 +110,7 @@ class token_iterator {
   simdjson_really_inline uint32_t peek_index(token_position position) const noexcept;
 
   const uint8_t *buf{};
-  token_position index{};
+  token_position _position{};
```

Comment:
Bikeshed nitpicking... I am not a fan of `_position` (as opposed to `position`) outside a language like Python where underscores have a clear meaning. Moreover, here you have another attribute called `buf` and not `_buf`.

One way to stress that it is an attribute is to do `this->position`. Some people hate that, of course.

Reply:
Of course, here you can't use `position`  because you have `position()`.

Reply:
I'm all ears for suggestions. I am not a fan of `_position` either. But the `position()` problem is more about the conflict with the method of the same name than anything else. I've been trying to put these things behind methods, since there is a statistically significant chance the implementation will actually change (either because the data within moves elsewhere, or we implement streamed parsing, or something else). I can remove `_` here and change callers because this is an internal class, but the principle still needs some thought.

For consistency, yeah, I should change the other one to `_buf`.

The only alternatives I've thought of are:
* `m_buf`. I have a violent reaction to Hungarian Notation, but maybe it's better here?
* Clever alternative names like `original_buf`. I don't like having two names for the same thing, generally.

Reply:
Note that I have a general policy of not spending too much time on bikeshedding. So... whatever... I really don't care how private variables are named in the end.